versions/development/SPEC.md

```diff
@@ -834,158 +833,63 @@ task t {
 
 ### Command Section
 
-The `command` section is the *task section* that starts with the keyword 'command', and is enclosed in either curly braces `{ ... }` or triple angle braces `<<< ... >>>`.
-It defines a shell command which will be run in the execution environment after all of the inputs are staged and before the outputs are evaluated.
-The body of the command also allows placeholders for the parts of the command line that need to be filled in.
-
-Expression placeholders are denoted by `${...}` or `~{...}` depending on whether they appear in a `command { }` or `command <<< >>>` body styles.
+The `command` section is the *task section* that starts with the keyword 'command', and is triple angle braces `<<< ... >>>`.
```

Comment:
Is this suggesting that `command { }` be deprecated in favor of `command <<< ....>>>`? Additionally, what type of expresison placeholders are you proposing?

Reply:
This is a mistake, i do think that but it should not be in this PR.